### DISCUSSION TITLE
Does Reindexing without "complete rescan" find updated metadata?

### DISCUSSION THREAD

**Discussion by marcelmindemann** [photoprism/photoprism#3859] [Q&A]
Hey, since Photoprism does not currently write back edited metadata to the EXIF header of the pictures, I am editing metadata via DigiKam. When I update some photo's metadata, say I add location data, after they had already been indexed, will running an Indexing job find the updated metadata? The documentation says that "Complete rescan" will re-index _already indexed and unchanged files_, but I don't know if the opposite true - does a normal index operation find changed files? If yes, how does detection work? Does it look at file modification dates?

**Comment by lastzero** [comment]
Yes it does, unless you've somehow disabled filesystem modification time updates so that it looks like the file wasn't changed.
